cmd/cluster-proxy/main.go

```diff
@@ -23,6 +23,7 @@ import (
 func main() {
 	rand.Seed(time.Now().UTC().UnixNano())
 
+	klog.InitFlags(nil)
```

Comment:
why we need this?

Reply:
https://github.com/kubernetes/klog/blob/main/README.md#how-to-use-klog
```
Use klog.InitFlags(nil) explicitly for initializing global flags as we no longer use init() method to register the flags
```

This is to enable different verbosity setting.